pattoo/db/schema/language.py

```diff
@@ -37,3 +38,7 @@ class Meta:
 
         model = LanguageModel
         interfaces = (graphene.relay.Node,)
+
+class ProtectedLanguage(graphene.Union):
```

Comment:
Add a docstring to explain what you are trying to do here. It will help with maintainability.

Reply:
@Dev-Dominic